operator/v1/0000_50_ingress-operator_00-ingresscontroller.crd.yaml

```diff
@@ -261,6 +261,104 @@ spec:
+                      properties:
+                        request:
+                          description: request is a list of HTTP request headers to set or delete. The actions either Set or Delete will be performed on the headers in sequence as defined in the list of request headers.
+                          items:
+                            description: IngressControllerHTTPHeader specifies configuration for setting or deleting an HTTP header.
+                            properties:
+                              action:
+                                description: actions specifies actions to perform on headers, such as setting or deleting headers.
+                                properties:
+                                  set:
+                                    description: set specifies how the HTTP header should be set. If this field is empty, no action is taken.
+                                    properties:
+                                      value:
+                                        description: value specifies a header value. Dynamic values can be added. The value will be interpreted as an HAProxy format string as defined in http://cbonte.github.io/haproxy-dconv/2.6/configuration.html#8.2.6 and may use HAProxy's %[] syntax and otherwise must be a valid HTTP header value as defined in https://datatracker.ietf.org/doc/html/rfc7230#section-3.2
+                                        maxLength: 4096
+                                        minLength: 1
+                                        pattern: ^(%(%|(\{[-+]?[QXE](,[-+]?[QXE])*\})?([A-Za-z]+|\[[.0-9A-Z_a-z]+(\([^)]+\))?(,[.0-9A-Z_a-z]+(\([^)]+\))?)*\]))|[^%[:cntrl:]])*$
+                                        type: string
+                                        x-kubernetes-validations:
+                                          - message: converter specified not permitted.
+                                            rule: '!(self.contains(''debug('') || self.contains(''json_query('') || self.contains(''set-var('') || self.contains(''map_str('') || self.contains(''map_str_int('') || self.contains(''map_str_ip('') || self.contains(''map_beg('') || self.contains(''map_beg_int('') || self.contains(''map_end_ip('') || self.contains(''map_sub('') || self.contains(''map_sub_int('') || self.contains(''map_sub_ip('') || self.contains(''map_dir('') || self.contains(''map_dir_int('') || self.contains(''map_dir_ip('') || self.contains(''map_dom('') || self.contains(''map_dom_int('') || self.contains(''map_dom_ip('') || self.contains(''map_end('') || self.contains(''map_end_int('') || self.contains(''map_end_ip('') || self.contains(''map_reg('') || self.contains(''map_reg_int('') || self.contains(''map_reg_ip('') || self.contains(''map_regm('') || self.contains(''map_int('') || self.contains(''map_int_int('') || self.contains(''map_int_ip('') || self.contains(''map_ip('') || self.contains(''map_ip_int('') || self.contains(''map_ip_ip(''))'
```

Comment:
This string looks weird because of all the single quotes, being wrapped by an outer single quote, might need to change the quoting on this